workflow/rules/preprocess/power_preprocess_all.smk

```diff
@@ -5,6 +5,4 @@ Process all files
 
 rule preprocess_all:
     input:
-        rules.process_gridfinder.output,
-        GENERATORS_LINES_CONSUMERS_ALL_BOXES,
```

Comment:
There's not much left in here now. Do we still want this rule? I think it was like a kind of intermediate target rule, a collection of expands to do all the preprocessing. I think I'd do away with these kinds of rules if possible. Have everything run per box as far through the workflow as possible, then have a target rule at the end.